src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
@markcowl,

The breaks in Powershell API due to this change will be handled in a separate pull request. I'll send out a pull request with the corresponding changes to powershell, shortly. It is **not** a breaking change in as far as powershell is concerned.

Powershell seems to be taking a dependency on SDK via nuget packages (and the corresponding dll reference). Merging in this change should not affect powershell unless we publish the SDK nuget pkg and consume it in powershell code; and release a newer version of powershell.